mica/archive/update_ocat_target_table.py

```diff
@@ -0,0 +1,89 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Update a mica Ska target table
+"""
+import argparse
+from pathlib import Path
+from astropy.table import Table
+import requests
+import numpy as np
+
+from ska_helpers.retry import retry_call
+from mica.common import MICA_ARCHIVE
+
+URL_OCATDETAILS = "https://cda.harvard.edu/srservices/ocatDetails.do?format=text"
+
+def get_options():
+    parser = argparse.ArgumentParser(
+        description="Update target table")
+    parser.add_argument("--datafile",
+                        default='target_table.h5')
+    opt = parser.parse_args()
+    return opt
+
+
+def main():
+    opt = get_options()
+    datafile = Path(opt.datafile)
```

Comment:
Better to put data type munging into the lower level function, that way someone could pass a string to `update_table` and it would work.